app/uk/gov/hmrc/teamsandrepositories/models/BuildJob.scala

```diff
@@ -57,7 +57,10 @@ sealed trait JenkinsObject
 
 object JenkinsObject {
   case class Folder(service: String, jenkinsURL: String, objects: Seq[JenkinsObject]) extends JenkinsObject
-  case class BuildJob(service: String, jenkinsURL: String, latestBuild: Option[BuildData]) extends JenkinsObject
+  case class BuildJob(service: String,
+                      jenkinsURL: String,
+                      latestBuild: Option[BuildData],
+                      gitHubUrl: Option[String]) extends JenkinsObject
```

Comment:
Wondering if this should be optional if it's effectively our id - if we can't get it, should we just fail the parse, and throw the result away?
This makes our model simpler (and collection joins etc.)

Reply:
Then if this is our id, we can move to first position in the model to make this clear.

Reply:
As discussed, `service` is the id (although `jobName` would be better named), and githubUrl is our identifier to `GithubRepository`

Reply:
@colin-lamed It is optional because there are build jobs (currently 47) that are not linked to a repository, for example the create-a-test-repository job.

Reply:
Are those jobs useful in catalogue?